app/services/books_api_services.rb

```diff
@@ -1,17 +1,26 @@
 class BooksApiServices
-  def get_users_books(user_id)
-    response = Faraday.get('heroku_jdgngskjdnfik/bookshelves') do |req|
-      req.params['user_id'] = user_id
+  def self.get_search_books(search)
+    Rails.cache.fetch("#{search}_book_search", expires_in: 6.hours) do
+      response = conn.get('/api/v1/book/search') do |req|
+        req.params[:search] = search
+      end
+      
+      JSON.parse(response.body, symbolize_names: true)
     end
-    #place holders here, may be unneccesary but wanted to have it good to go in the event it is neccessary
+  end
 
+  def self.get_a_book(volume_id)
+    response = conn.get('/api/v1/book/search') do |req|
```

Comment:
I think this will actually be a different endpoint to return book details - if the book is in library